### PR TITLE
Edisontest

### PR DESCRIPTION
**Pull request by IversonHan**
unreachable arg

### CODE REVIEW

File: contracts/scripts/deploy-erc20.ts [hunk 1]
```diff
@@ -59,7 +59,7 @@ async function main() {
 
     program
         .command('add')
-        .option('-n, --token-name <token_name>')
+        .option('-n, --token_name <token_name>')
```
Comment (ly0va) [inline contracts/scripts/deploy-erc20.ts:62]: Why this change? It breaks [this line](https://github.com/matter-labs/zksync/blob/f958bd54000c5203ed54b55dee073d5b03f192be/infrastructure/zk/src/run/run.ts#L29)

Comment (IversonHan) [reply contracts/scripts/deploy-erc20.ts:62]: Hey boy, the command does't work with option token-name, the process cannot receive this parameter unless they are both token_name or token-name.I hope you will test it!

> Why this change? It breaks [this line](https://github.com/matter-labs/zksync/blob/f958bd54000c5203ed54b55dee073d5b03f192be/infrastructure/zk/src/run/run.ts#L29)

Comment (ly0va) [reply contracts/scripts/deploy-erc20.ts:62]: According to many examples in [Commander readme](https://github.com/tj/commander.js#readme) that's not true.
And the line would still break. Either it should be fixed, or [this line](https://github.com/matter-labs/zksync/blob/f958bd54000c5203ed54b55dee073d5b03f192be/contracts/scripts/deploy-erc20.ts#L70) should be changed to `name: cmd.tokenName`.

Comment (IversonHan) [reply contracts/scripts/deploy-erc20.ts:62]: > According to many examples in [Commander readme](https://github.com/tj/commander.js#readme) that's not true. And the line would still break. Either it should be fixed, or [this line](https://github.com/matter-labs/zksync/blob/f958bd54000c5203ed54b55dee073d5b03f192be/contracts/scripts/deploy-erc20.ts#L70) should be changed to `name: cmd.tokenName` (with the previous commit reverted).

I'm sorry, this line[                name: cmd.token_name,](url) is the source of the error.So i changed it to --token_name.

**Comment by ly0va** [comment]
@IversonHan Thanks for your contribution, I cherry-picked your commit into another pull request.
Closing in favor of #369
